src/Plugin.php

```diff
@@ -69,11 +69,14 @@ public static function init() {
     add_filter('pre_option_default_ping_status', '__return_zero');
     add_filter('pre_option_default_pingback_flag', '__return_zero');
 
+    // Disable pingback functionality and remove x-pingback HTTP header.
```

Comment:
1. "X-Pingback" should be written in the lettercasing as it appears.

2. This comment appears to be a heading for all three filters that follow, not just the first one (the first filter is only about the HTTP header).  In closely connected cases like this, You can combine the other two comments into this first one as necessary (still retaining relevant details) and put all three filter registrations directly below this single comment.